### ISSUE TITLE
Some Questions about the code

### ISSUE THREAD

**Issue by lylaiyy** [FLEECERmw/PrivacyFedLLM#1]
Thanks for the author's open source code. But I have some questions I hope to get your reply.
1. in the processing of Dolly dataset, the code directly removes the information of "context" field. Is this normal?
2. in my own experiments, I found that fedllm is very easy to over fit. In this code, I observed that the training loss can be almost reduced to 0. I think this is a high probability that fitting will occur on the verification set, although the code does not provide the verification set part. I want to know whether the author has encountered the problem of fitting, and how to solve it?

**Comment by SII-FLEEECERmw** [comment]
Hi, thanks for your question. 
1, When I was processing the dolly-15k dataset, I found that the 'context' column was not provided for a large portion of the samples. For those provided, the content was often highly duplicated with the 'response', and those questions were usually common world knowledges. Given this, I decided to simply discard it. You are welcome to add it back as desired by modifying the script in data/load_data.py.

2, I think overfitting largely depends on the specific experiment settings and data scale, you might try lowering the rank, increasing the lora_dropout value or reducing the number of local epochs and learning rate. Grid search might help this.

For evaluating performance, I use the lm-eval-harness repository after training is complete.

**Comment by lylaiyy** [comment]
Thank you for your responses!
